### PR TITLE
Add set up DeFi tools section

### PR DESCRIPTION
**Pull request by annamehr**

### CODE REVIEW

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+1. Go to https://metamask.io/download.html.
+2. Choose the browser you want to use.
+3. You will be directed to corresponding web store.
+4. Follow the steps there to add the extension to your browser.
+5. After setting up your wallet, Metamask should pop-up as a Fox icon ![](https://storage.googleapis.com/centrifuge-hackmd/upload_7dd07df8a7fc7d1b29743ec9535dbba7.png =20x) in your browser bar.
+
+### How to use Metamask with Tinlake
+- Connect your account.
+- Click on Connect (top right corner).
+- Choose the Metamask or the wallet you would like to connect.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:21]: - Choose the MetaMask wallet in the popup

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### How to install Metamask in your browser
+1. Go to https://metamask.io/download.html.
+2. Choose the browser you want to use.
+3. You will be directed to corresponding web store.
+4. Follow the steps there to add the extension to your browser.
+5. After setting up your wallet, Metamask should pop-up as a Fox icon ![](https://storage.googleapis.com/centrifuge-hackmd/upload_7dd07df8a7fc7d1b29743ec9535dbba7.png =20x) in your browser bar.
+
+### How to use Metamask with Tinlake
+- Connect your account.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:19]: How about removing this bullet and turning it into the heading above? I.e.
### Connect MetaMask with Tinlake

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+id: set-up-defi-tools
+order: 12
+title: Set up DeFi Tools
+---
+
+## Setting up Metamask
+### What is Metamask?
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:8]: MetaMask

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+id: set-up-defi-tools
+order: 12
+title: Set up DeFi Tools
+---
+
+## Setting up Metamask
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:7]: ## Setting up a MetaMask browser wallet

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+4. Follow the steps there to add the extension to your browser.
+5. After setting up your wallet, Metamask should pop-up as a Fox icon ![](https://storage.googleapis.com/centrifuge-hackmd/upload_7dd07df8a7fc7d1b29743ec9535dbba7.png =20x) in your browser bar.
+
+### How to use Metamask with Tinlake
+- Connect your account.
+- Click on Connect (top right corner).
+- Choose the Metamask or the wallet you would like to connect.
+![](https://storage.googleapis.com/centrifuge-hackmd/upload_7ce191ec0d5455f0228b8e9779c54719.png)
+
+## Set up a Ledger Hardware Wallet
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:24]: ## Setting up a Ledger hardware wallet

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+### How to use Metamask with a hardware wallet, e.g. Ledger?
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:25]: ## What is a hardware wallet?

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+A hardware wallet stores the user's private keys in a secure hardware device. To use an address/wallet stored on a hard ware wallet a user needs to be in physical posession of this hardware wallet. Popolar hardware wallets are e.g [Ledger](https://www.ledger.com/) or [Trezor](https://trezor.io/).
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:26]: To use an address stored in a hardware wallet a user needs to be in physical possession of the device. Popular hardware wallets are [Ledger](https://www.ledger.com/) or [Trezor](https://trezor.io/).

Comment (armandomistral) [reply docs/use/Set-Up-DeFi-Tools/index.md:26]: @annamehr, is Trezor also supported by Tinlake?

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+#### Set up the Ledger
+Download the [Ledger Live App](https://www.ledger.com/ledger-live/) and follow the steps in the App also described [here]( https://support.ledger.com/hc/en-us/articles/360018784134) to set up you Ledger.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:29]: Shortened, lowercase use of "app":
Download the [Ledger Live](https://www.ledger.com/ledger-live/) and follow the steps in the app described [here](https://support.ledger.com/hc/en-us/articles/360018784134)[](https://github.com/centrifuge/documentation/blob/81b4b97bddb0ffad2e530620d2f809c270949cc9/docs/use/Set-Up-DeFi-Tools/index.md#install-ethereum-app-on-ledger-device).

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+### Install Ethereum App on Ledger Device
+1. Open the Manager in Ledger Live.
+2. Connect and unlock your device.
+3. If asked, allow the manager on your device.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:34]: "Manager" (consistent wording)

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+![](https://storage.googleapis.com/centrifuge-hackmd/upload_7ce191ec0d5455f0228b8e9779c54719.png)
+
+## Set up a Ledger Hardware Wallet
+### How to use Metamask with a hardware wallet, e.g. Ledger?
+A hardware wallet stores the user's private keys in a secure hardware device. To use an address/wallet stored on a hard ware wallet a user needs to be in physical posession of this hardware wallet. Popolar hardware wallets are e.g [Ledger](https://www.ledger.com/) or [Trezor](https://trezor.io/).
+
+#### Set up the Ledger
+Download the [Ledger Live App](https://www.ledger.com/ledger-live/) and follow the steps in the App also described [here]( https://support.ledger.com/hc/en-us/articles/360018784134) to set up you Ledger.
+
+### Install Ethereum App on Ledger Device
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:31]: Sentence case:
### Install Ethereum app on Ledger device

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+- Connect your account.
+- Click on Connect (top right corner).
+- Choose the Metamask or the wallet you would like to connect.
+![](https://storage.googleapis.com/centrifuge-hackmd/upload_7ce191ec0d5455f0228b8e9779c54719.png)
+
+## Set up a Ledger Hardware Wallet
+### How to use Metamask with a hardware wallet, e.g. Ledger?
+A hardware wallet stores the user's private keys in a secure hardware device. To use an address/wallet stored on a hard ware wallet a user needs to be in physical posession of this hardware wallet. Popolar hardware wallets are e.g [Ledger](https://www.ledger.com/) or [Trezor](https://trezor.io/).
+
+#### Set up the Ledger
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:28]: Should this not be the same heading as the other sections? i.e.
### Set up the Ledger device

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+Download the [Ledger Live App](https://www.ledger.com/ledger-live/) and follow the steps in the App also described [here]( https://support.ledger.com/hc/en-us/articles/360018784134) to set up you Ledger.
+
+### Install Ethereum App on Ledger Device
+1. Open the Manager in Ledger Live.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:32]: How about adding an introduction sentence on top of the steps and removing the "See also…" part at the end:
set up your first Ledger Ethereum (ETH) account.
"Set up your first Ledger Ethereum account. See more [here](https://support.ledger.com/hc/en-us/articles/360009576554-Ethereum-ETH-?docs=true)

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+3. If asked, allow the manager on your device.
+4. Find Ethereum in the app catalog.
+5. Click the Install button of the app.
+    - An installation window appears
+    - Your device will display `Processing...`
+    - The app installation is confirmed
+
+See also [Install Ethereum App on your Ledger](https://support.ledger.com/hc/en-us/articles/360009576554-Ethereum-ETH-).
+
+### Configure Ethereum App
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:43]: ### Configure Ethereum app

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+5. Click the Install button of the app.
+    - An installation window appears
+    - Your device will display `Processing...`
+    - The app installation is confirmed
+
+See also [Install Ethereum App on your Ledger](https://support.ledger.com/hc/en-us/articles/360009576554-Ethereum-ETH-).
+
+### Configure Ethereum App 
+
+1. Open the Ethereum App Settings.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:45]: Open the app **Settings**.

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+2. Change Contract Data to `Yes`.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:46]: 2. Change **Contract Data** to `Yes`.

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+3. [OPTIONAL] For older versions of Ledger, make sure Browser Support is set to `Yes`. Newer Ledger Versions are supporting it by default.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:47]: **Optional**: For older versions of Ledger, make sure **Browser Support** is set to `Yes`. Newer Ledger versions are supporting it by default.

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+
+See also [Install Ethereum App on your Ledger](https://support.ledger.com/hc/en-us/articles/360009576554-Ethereum-ETH-).
+
+### Configure Ethereum App 
+
+1. Open the Ethereum App Settings.
+2. Change Contract Data to `Yes`.
+3. [OPTIONAL] For older versions of Ledger, make sure Browser Support is set to `Yes`. Newer Ledger Versions are supporting it by default. 
+
+### Connect Hardware Wallet in Metamask
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:49]: ### Connect hardware wallet in MetaMask

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+1. Open Metamask by clicking on the fox icon.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:50]: MetaMask

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+2. Make sure to select the correct network (`Main Ethereum Network` or `Kovan Test Network`) at the top.
+3. Click on the circle in the top right corner.
+4. Click on `Connect Hardware Wallet`.
+5. A new modal opens. Select `Ledger` and click `Connect`.
+6. Select address you would like to use. Your Ledger already entails several addresses you can chose from (autogenerated for you).
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:55]: Select the address you would like to use. Ledger already offers several autogenerated addresses to choose from.

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+
+### Confirm transactions with your hardware wallet (Ledger/Trezor)
+When your Metamask account is connected to your hardware wallet then you need to confirm each transaction with the ledger.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:58]: When your MetaMask account is connected to the hardware wallet you need to confirm each transaction with Ledger.

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 21]
```diff
@@ -0,0 +1,62 @@
...
+2. Make sure to select the correct network (`Main Ethereum Network` or `Kovan Test Network`) at the top.
+3. Click on the circle in the top right corner.
+4. Click on `Connect Hardware Wallet`.
+5. A new modal opens. Select `Ledger` and click `Connect`.
+6. Select address you would like to use. Your Ledger already entails several addresses you can chose from (autogenerated for you).
+
+### Confirm transactions with your hardware wallet (Ledger/Trezor)
+When your Metamask account is connected to your hardware wallet then you need to confirm each transaction with the ledger. 
+
+- Connect ledger with your computer.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:60]: Connect the Ledger with your computer.

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 22]
```diff
@@ -0,0 +1,62 @@
...
+- Issue the transaction.
+- Confirm transaction with hardware wallet.
```
Comment (armandomistral) [inline docs/use/Set-Up-DeFi-Tools/index.md:62]: Confirm the transaction with the hardware wallet.

**Review by armandomistral — COMMENTED** [review]
Thanks, @annamehr, very helpful documentation! Reminds me of the guide I used to get started (https://thedefiant.io/getting-started-with-defi/). Suggesting using sentence case for heading (until we have a style manual)

**Review by tsx49 — APPROVED** [review]

**Review by tsx49 — COMMENTED** [review]
Fixed metamask logo image and changed last bullet points to numbers

**Comment by annamehr** [comment]
> 

changed it to sentence case

File: docs/use/Set-Up-DeFi-Tools/index.md [hunk 23]
```diff
@@ -0,0 +1,62 @@
+---
+id: set-up-defi-tools
+order: 12
+title: Set up DeFi Tools
+---
+
+## Setting up Metamask
+### What is Metamask?
+MetaMask is a browser extension that allows you to use Ethereum Protocols or `dApps` ("decentralized Apps") such as Tinlake right in your browser. Metamask holds your private keys, and once it's set up, you can connect it to the dApp in your browser and confirm your transactions in the dApp with Metamask. It also serves as a wallet for ETH and `ERC20 tokens`, such as DAI, TIN or DROP and allows you to transfer and receive such tokens.
```
[the remark this answers is not in the document]
Comment (annamehr) [reply docs/use/Set-Up-DeFi-Tools/index.md:9]: done

**Review by lucasvo — APPROVED** [review]

**Pull request merged**